packages/mail-client/src/socket-proxy.ts

```diff
@@ -18,8 +27,9 @@ export async function customAuthRequestHandler(jwt: string): Promise<Buffer> {
     // The actual value of the JWT in its encoded form
     ...jwtBytes,
     // Length of the Cloud Connector location ID (0 if no Cloud Connector location ID is used)
-    0x00
+    0x00,
```

Comment:
[q] Shouldn't the length also be adjusted?

Reply:
oh, duh, yeah, it probably should 🤦 thanks